.github/pull_request_template.md

```diff
@@ -0,0 +1,30 @@
+## Status
+**READY/IN DEVELOPMENT/HOLD**
+ - [ ] Ready
+ - [ ] In development
+ - [ ] Hold
+
+## Closes issue(s)
+A list of issues that are closed by this PR
```

Comment:
I think we can move this to a todo to make sure people are connecting the PR with the issues they are working on.